tests/unit/util/main.c

```diff
@@ -288,6 +288,37 @@ static void test_FOR_EACH_IDX_FIXED_ARG(void)
 	zassert_equal(a2, 3, "Unexpected value %d", a2);
 }
 
+static void test_IS_EMPTY(void)
+{
+	#define test_IS_EMPTY_REAL_EMPTY
+	#define test_IS_EMPTY_NOT_EMPTY XXX_DO_NOT_REPLACE_XXX
+	zassert_true(IS_EMPTY(test_IS_EMPTY_REAL_EMPTY),
+		     "Expected to be empty");
+	zassert_false(IS_EMPTY(test_IS_EMPTY_NOT_EMPTY),
+		      "Expected to be non-empty");
+}
+
+static void test_LIST_DROP_EMPTY(void)
+{
+	/*
+	 * The real definition should be:
+	 *  #define TEST_BROKEN_LIST ,Henry,,Dorsett,Case,
+	 * but checkpatch complains, so below equivalent is defined.
```

Comment:
just let checkpatch complain, maintainers have the discretion to ignore it. I'd value the extra robustness.